tests/unittests/test_ds_identify.py

```diff
@@ -705,8 +721,8 @@ def _print_run_output(rc, out, err, cfg, files):
         '-- cfg --', util.json_dumps(cfg)]))
     print('-- files --')
     for k, v in files.items():
-        if "/_shwrap" in k:
-            continue
+        #if "/_shwrap" in k:
```

Comment:
re-enable. the output of _shwrap is a bit over the top. i just put it here so you could debug more easily.

it'll also fix you pycodestyle error ;)